util/dvsim/Scheduler.py

```diff
@@ -48,9 +48,12 @@ def get_next_item(arr, index):
 class Scheduler:
     '''An object that runs one or more Deploy items'''
 
-    def __init__(self, items, launcher_cls):
+    def __init__(self, items, launcher_cls, interactive):
         self.items = items
 
+        # Interactive mode
+        self.interactive = interactive
```

Comment:
Seems like this instance variable is unused.

Reply:
removed.